coderd/database/queries/organizationmembers.sql

```diff
@@ -1,13 +1,29 @@
--- name: GetOrganizationMemberByUserID :one
+-- name: OrganizationMembers :many
+-- Arguments are optional with uuid.Nil to ignore.
+--  - Use just 'organization_id' to get all members of an org
+--  - Use just 'user_id' to get all orgs a user is a member of
+--  - Use both to get a specific org member row
 SELECT
-	*
+	sqlc.embed(organization_members),
+	users.username
 FROM
 	organization_members
+		INNER JOIN
+	users ON organization_members.user_id = users.id
 WHERE
-	organization_id = $1
-	AND user_id = $2
-LIMIT
-	1;
+	true
```

Comment:
just for my understanding, why do we need to do `where true and case` instead of a `where case`?

Reply:
You are right, we don't need the `true and` :+1: